app/views/shared/results/_search_results.html.haml

```diff
@@ -18,7 +18,7 @@
         with dates
       - if provider_facets.present?
         from
-        = "#{ 'over' if provider_facets.count >= 2000 }"
+        = "#{ 'over' if provider_facets.count >= 240 }"
```

Comment:
From a UX standpoint, this is a bit awkward.  Most general queries will return far more than 240 providers (for example, a search for "George Washington" returns 633).  And 240 seems a strange number to broadcast to the user.  I would vote for removing this user message altogether until we can come up with a better way of constructing it, and leaving it at just the total number of search results -- so essentially removing lines 19-27.